src/ui/api/project.go

```diff
@@ -120,24 +132,15 @@ func (p *ProjectAPI) Post() {
 	}
 
 	go func() {
-		user, err := dao.GetUser(models.User{
-			UserID: p.userID,
-		})
-		if err != nil {
-			log.Errorf("failed to get user by ID %d: %v", p.userID, err)
-			p.CustomAbort(http.StatusInternalServerError, http.StatusText(http.StatusInternalServerError))
-		}
-
-		accessLog := models.AccessLog{
-			Username:  user.Username,
-			ProjectID: projectID,
-			RepoName:  project.Name + "/",
-			RepoTag:   "N/A",
-			GUID:      "N/A",
-			Operation: "create",
-			OpTime:    time.Now(),
-		}
-		if err = dao.AddAccessLog(accessLog); err != nil {
+		if err = dao.AddAccessLog(
+			models.AccessLog{
+				Username:  p.SecurityCtx.GetUsername(),
+				ProjectID: projectID,
+				RepoName:  pro.ProjectName + "/",
+				RepoTag:   "N/A",
+				Operation: "create",
+				OpTime:    time.Now(),
+			}); err != nil {
 			log.Errorf("failed to add access log: %v", err)
 			p.CustomAbort(http.StatusInternalServerError, http.StatusText(http.StatusInternalServerError))
```

Comment:
I find here is not consistent with other parts that create access logs, probably should not return 500. just log the error.